integration/Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN mv ./kubectl /usr/local/bin
 
 COPY bin/kubectl-gadget /bin/kubectl-gadget
 COPY --from=build_base /build/gadget-integration.test /bin/gadget-integration.test
-CMD ["/bin/sh", "-c", "gadget-integration.test -integration -test.timeout 20m"]
+CMD ["/bin/sh", "-c", "gadget-integration.test -test.v -integration -test.timeout 20m"]
```

Comment:
Oh sorry, I wasn't aware of that discussion. However, I don't see the errors you both mentioned there. Maybe I missed something but I don't expect that a text printed with `t.Logf` will end up in the raw log with `##[error]` as the prefix.

In addition, during work I have been doing to run the integration tests in ARO, I found really useful to also have the output of the successful gadgets to understand if similar gadgets to the one that is failing are working correctly or not.

cc @mauriciovasquezbernal

Reply:
>Oh sorry, I wasn't aware of that discussion.

No problem, you cannot know every discussion.

> However, I don't see the errors you both mentioned there.

I cannot remember exactly, but I think some message using `t.Log` or something like this were printed in red within the CI.

> In addition, during work I have been doing to run the integration tests in ARO, I found really useful to also have the output of the successful gadgets to understand if similar gadgets to the one that is failing are working correctly or not.

I understand your point of view.
I am not against adding some prints, but I prefer to not print within the CI.
When testing locally, we are then free to add this flag if we find it useful.

Reply:
> However, I don't see the errors you both mentioned there

That's right, I don't know why those are not printed now, in the past they were there -> https://github.com/kinvolk/inspektor-gadget/actions/runs/1439250610. 

I don't have a strong opinion on printing the output. My concern is that having the output of all tests (even the passing ones) makes it difficult to understand what's going on..

Reply:
Yeah, I agree it is difficult to understand. However, if integration-tests job successes, one doesn't even open the logs. Instead, if something failed, it will be very useful to have all the outputs there. IMO, I prefer having them there even if it is difficult to follow instead of don't have it at all and losing useful information to analyse the failure.

Reply:
> IMO, I prefer having them there even if it is difficult to follow instead of don't have it at all and losing useful information to analyse the failure.

The tests are normally independent from each other (expect block I/O ones), so I do not see how the output of one test can help you debugging one which is failing.

Reply:
> The tests are normally independent from each other (expect block I/O ones), so I do not see how the output of one test can help you debugging one which is failing.
>
Not for the output of the gadget itself but for the code they share.

I see my justifications don't convince you both 100% thus I will remove this change for now. Let's see if you find it useful in the future.

Reply:
>Not for the output of the gadget itself but for the code they share.

You want to check functions like `RunTraceAndPrintStream()` works, right?

> I see my justifications don't convince you both 100% thus I will remove this change for now. Let's see if you find it useful in the future.

Even though I am not convinced, I do not have a strong opinion.
If having the output really helps you, then I think it would be helpful for you and then for us.